app/models/nic/base.rb

```diff
@@ -178,6 +178,12 @@ def ip6=(addr)
       super(Net::Validations.normalize_ip6(addr))
     end
 
+    def matches_subnet?(ip_field, subnet_field)
+      return unless send(subnet_field).present?
+      ip_value = send(ip_field)
+      value.present? && public_send(subnet_field).contains?(ip_value)
```

Comment:
`value` here has to be changed to `ip_value`, this is causing a bunch of tests to fail